src/main.js

```diff
@@ -24,6 +34,7 @@ new Vue({
     el: '#app',
     router,
     store,
+    usersStore,
```

Comment:
wrong place, remove this userStore from here. 
I think you are already using usersStore in admin/store

Reply:
yes